test/scripts/bridge/set-env.sh

```diff
@@ -22,9 +22,9 @@ export contract_dir="$relayer_root_dir/snowbridge/contracts"
 test_helpers_dir="$web_dir/packages/test-helpers"
 relay_bin="$relayer_root_dir/build/tanssi-bridge-relayer"
 
-RELAYER_COMMIT="05b5e6cf8fe836690cca4e88d2dff3307bf17fa4" # TODO: Change to tag when we do releases
+RELAYER_COMMIT="7885754f19a4deababd24f92737fe94935e9af5a" # TODO: Change to tag when we do releases
```

Comment:
This commit refers to your branch right? If yes, this means we will have to merge the PRs in other repo before we can merge this one.